### ISSUE TITLE
add Parsers

### ISSUE THREAD

**Issue by Creckeryop** [Creckeryop/NOBORU-parsers#1]

**Comment by Creckeryop** [comment]
Added InManga fff736a1e2cfa0d0da81849b3ae1b06ab387c5e5

**Comment by eze92** [comment]
Hi , Could you add this parsers https://tmofans.com/ in future update ? Thanks

**Comment by Creckeryop** [comment]
> Hi , Could you add this parsers https://tmofans.com/ in future update ? Thanks for you job
I don't know how to work with sites that asks for a captcha, so i won't add it in near future, sorry

**Comment by eze92** [comment]
Could you add more parsers the manga in spanish ?

**Comment by Creckeryop** [comment]
> Could you add more parsers the manga in spanish ?
I'll be glad if you give me some links to sites i can write parser on

**Comment by eze92** [comment]
I leave you a pair that I found in case you can add it

http://es.ninemanga.com/category/updated.html

https://mangalibre.com/

https://submangas.net/

http://www.animextremist.com/

**Comment by Creckeryop** [comment]
> I leave you a pair that I found in case you can add it
> 
> http://es.ninemanga.com/category/updated.html
> 
> https://mangalibre.com/
> 
> https://submangas.net/
> 
> http://www.animextremist.com/

Thanks for links, hope i will add some of them soon

**Comment by eze92** [comment]
you're welcome I'm glad if I can be of help. I pass you another that even has comics in Spanish
https://heavenmanga.com/
The most complete is the one that I had mentioned before
 https://tmofans.com/ but it has the catpcha problem ... thank you very much for your work

**Comment by Creckeryop** [comment]
> you're welcome I'm glad if I can be of help. I pass you another that even has comics in Spanish
> https://heavenmanga.com/
> The most complete is the one that I had mentioned before
>  https://tmofans.com/ but it has the catpcha problem ... thank you very much for your work

i don't have a lot of time to add sites, i will continue work next time

**Comment by qbaddev** [comment]
https://github.com/qbaddev/simple-nudemoon-api

**Comment by Creckeryop** [comment]
Ну я не знаю как этим пользоваться, спасибо

**Comment by qbaddev** [comment]
> Ну я не знаю как этим пользоваться, спасибо

Если есть надобность, могу помочь.

**Comment by Creckeryop** [comment]
Нет, ну скорее всего мне это не поможет, только в случае если эти папки будут на каком нибудь сервере, в лучшем случае на nudemoon, в любом случае я и так паршу nudemoon

**Comment by qbaddev** [comment]
> Нет, ну скорее всего мне это не поможет, только в случае если эти папки будут на каком нибудь сервере, в лучшем случае на nudemoon, в любом случае я и так паршу nudemoon, лан спасибо

Прости, не заметил. Удачи.

**Comment by ElRaru** [comment]
Can add the spanish NSFW  https://tmohentai.com/ page please?

**Comment by Creckeryop** [comment]
> Can add the spanish NSFW https://tmohentai.com/ page please?

This site uses capcha

**Comment by Creckeryop** [comment]
> Can add the spanish NSFW https://tmohentai.com/ page please?
Added [ES]DoujinHentai (NSFW)
Popular, Latest, Alphabet tabs shows same manga (so if site will fix it, they should work correctly)

**Comment by OM532** [comment]
https://mangalib.me/ ?

**Comment by Creckeryop** [comment]
> https://mangalib.me/ ?

Сайт использует какую-то супержесткую защиту, которую я даже не знаю как обойти, сорян

**Comment by OM532** [comment]
Печально.

**Comment by Waspin666** [comment]
Hi,
2 things about the parsers:
1. xoxocomics.com are now being detected as containing malicious software when entering link manually, most probably should not be any issue that may harm the parsing, but better to let know.
2. Would it be possible to add https://readcomiconline.to/ ? They seem to have quite a few things that are missing from anywhere else
Cheers!

**Comment by ordinary-smth** [comment]
https://readcomiconline.to/ ?

**Comment by Waspin666** [comment]
yes, on the first glance it is similar to readcomicsonline.ru, but actually the content is different

**Comment by Creckeryop** [comment]
@ordinary-smth @Waspin666, Added readcomiconline.to. Sorry for long waiting

**Comment by OM532** [comment]
https://mangaonelove.ru/ ?

**Comment by Creckeryop** [comment]
> https://mangaonelove.ru/ ?

Done

**Comment by eze92** [comment]
its posible https://www.kumanga.com/ ??

**Comment by Creckeryop** [comment]
> its posible https://www.kumanga.com/ ??

![image](https://user-images.githubusercontent.com/25725992/108261686-c8e56d00-7185-11eb-8720-b611a0737509.png)

**Comment by OM532** [comment]
https://mangahub.ru/
https://you-anime.ru/

**Comment by leandro-maciel** [comment]
https://tsukimangas.com/
https://mangalivre.net/

**Comment by DiegooBedin** [comment]
Hello, First of all, I wanted to say that this application is incredible, and I am a big fan of this wonderful work.

I saw that there are few parsers in Portuguese-BR or Portuguese in the application .. I tried to add some manually but I was not successful with my limited knowledge in language ..

Could you try to add some for us? I assure you that there are several manga fans around here, including hentai in Brazil and Portuguese countries. I separated some for you to see ..

https://www.brmangas.com/
https://mangalivre.net/
https://yesmangas1.com/
https://unionmangas.top/inicial
https://www.supermangas.site/
https://mundohentaioficial.com/
https://mangayabu.top/
https://muitomanga.com/
https://goldenmangas.top/
https://unionmangas.top/inicial
https://mangahosted.com/

Obviously, I don't use them all, but it is agreed that these are the best known here.

Tip: We Brazilians don't mind joining Pt-Br to Portuguese in the app as a single language, it is basically the same, with small differences, as with British and American English. Maybe it will help in something.

Anyway, thanks for your time and work.

**Comment by Creckeryop** [comment]
> Hello, First of all, I wanted to say that this application is incredible, and I am a big fan of this wonderful work.
> 
> I saw that there are few parsers in Portuguese-BR or Portuguese in the application .. I tried to add some manually but I was not successful with my limited knowledge in language ..
> 
> Could you try to add some for us? I assure you that there are several manga fans around here, including hentai in Brazil and Portuguese countries. I separated some for you to see ..
> 
> https://www.brmangas.com/
> https://mangalivre.net/
> https://yesmangas1.com/
> https://unionmangas.top/inicial
> https://www.supermangas.site/
> https://mundohentaioficial.com/
> https://mangayabu.top/
> https://muitomanga.com/
> https://goldenmangas.top/
> https://unionmangas.top/inicial
> https://mangahosted.com/
> 
> Obviously, I don't use them all, but it is agreed that these are the best known here.
> 
> Tip: We Brazilians don't mind joining Pt-Br to Portuguese in the app as a single language, it is basically the same, with small differences, as with British and American English. Maybe it will help in something.
> 
> Anyway, thanks for your time and work.

Added BRMangas

unionmangas.top is already in Noboru
i really can't open most of sites because of Access denied error

i will try to add https://www.supermangas.site/ and https://mangayabu.top/

**Comment by leandro-maciel** [comment]
> > Hello, First of all, I wanted to say that this application is incredible, and I am a big fan of this wonderful work.
> > I saw that there are few parsers in Portuguese-BR or Portuguese in the application .. I tried to add some manually but I was not successful with my limited knowledge in language ..
> > Could you try to add some for us? I assure you that there are several manga fans around here, including hentai in Brazil and Portuguese countries. I separated some for you to see ..
> > https://www.brmangas.com/
> > https://mangalivre.net/
> > https://yesmangas1.com/
> > https://unionmangas.top/inicial
> > https://www.supermangas.site/
> > https://mundohentaioficial.com/
> > https://mangayabu.top/
> > https://muitomanga.com/
> > https://goldenmangas.top/
> > https://unionmangas.top/inicial
> > https://mangahosted.com/
> > Obviously, I don't use them all, but it is agreed that these are the best known here.
> > Tip: We Brazilians don't mind joining Pt-Br to Portuguese in the app as a single language, it is basically the same, with small differences, as with British and American English. Maybe it will help in something.
> > Anyway, thanks for your time and work.
> 
> Added BRMangas
> 
> unionmangas.top is already in Noboru
> i really can't open most of sites because of Access denied error
> 
> i will try to add https://www.supermangas.site/ and https://mangayabu.top/

Can you try to add https://tsukimangas.com/ too?

**Comment by eze92** [comment]
https://www.mangatigre.com/

**Comment by Creckeryop** [comment]
> https://www.mangatigre.com/

Site is protected by tokens, and i really dunno how to add it to NOBORU

**Comment by Creckeryop** [comment]
> https://mangahub.ru/
> https://you-anime.ru/

Added mangahub ru https://github.com/Creckeryop/NOBORU-parsers/commit/f5d1115880affda3f88be33a13feae6c2914db2c

**Comment by Hikikomori15** [comment]
Manganelo has moved from .com to .tv or altenative manganato.com and is not working anymore on noboru

**Comment by Creckeryop** [comment]
> Manganelo has moved from .com to .tv or alternative manganato.com and is not working anymore on noboru

what is manganato.com?

**Comment by Hikikomori15** [comment]
> > Manganelo has moved from .com to .tv or alternative manganato.com and is not working anymore on noboru
> 
> what's better manganato.com or manganelo.tv?

I don't think there is really a difference. It's probably the same site with different url. Choosing manganelo.tv would probably prevent confusion because its the same url only with .tv

**Comment by Roberto20012006** [comment]
Can you add this one please
Name: Truyengihot
Link: https://www.truyengihot.vip/
Language: Vietnamese

**Comment by Creckeryop** [comment]
> Can you add this one please Name: Truyengihot Link: https://www.truyengihot.vip/ Language: Vietnamese

Done (search doesn't work on site, so in Noboru too, probably it will work on Noboru when site will be updated)

**Comment by Roberto20012006** [comment]
Thank you so much
I found another site of this working so can you replace it in please
Name: Truyengihot+
Link: https://truyengihot.net/
Language: Vietnamese

Thank you a lot OwO

**Comment by OmaGamer** [comment]
Hi,
please add a Araibc website. I cloudn't do it myself just like ASimYi.
Here is one Arabic manga website

Name: Manga on line (مانجا اون لاين)
Link: https://onma.me/
Language: Arabic

Thanks for this App it's great!

**Comment by toprak** [comment]
komga would be awesome

**Comment by Creckeryop** [comment]
@OmaGamer, added Onma, check how it works

**Comment by Creckeryop** [comment]
@toprak 
You mean create a media server?
I thought about that idea but I really don't have too much time to support my project.

**Comment by toprak** [comment]
no I meant if someone has create media server (komga), noboru should acces to its server

**Comment by OmaGamer** [comment]
@Creckeryop thanks it works but it now says that the Domain is going to be changed soon. The new one is onma.tom

**Comment by Creckeryop** [comment]
@OmaGamer Maybe that's hard to read, there's no in-app arabic language support, and I really sorry about that

**Comment by toprak** [comment]
@Creckeryop 
```Also I don't need any law problems, my app is just a player and I don't host any websites with images sweat_smile.```
Let me summarize it like this, then, I was talking about adding something like an extension for people who have a komga server on their computer, and there it should be like this, the user will set  the ip address of the Komga server from the extension settings and access it (by ip I mean local ip, like 192.168.1.5).

**Comment by PaxHadesTR** [comment]
Hi, can you add these websites:

https://mangasehri.com(TURKISH)
https://tempestfansub.com(TURKISH)

and MangaTR is already exist but not showing in catalog

**Comment by eze92** [comment]
https://mangas.in/

**Comment by Minhgiale1567** [comment]
@Creckeryop can you replace to this website https://m.truyenqqq.vn/ language Vietnamese, old website can not access anymore
Hope you replace soon

**Comment by eze92** [comment]
https://bokugents.com/

**Comment by zooki2006** [comment]
Sorry if this isn't the place for this, but if posable i would like to request http://webtoons.com/ since its the only thing stopping me from using this for all my online reading*
*aside from webcomics that only host on the one site.
